src/MOI_wrapper.jl

```diff
@@ -136,3 +136,14 @@ end
 function MOI.get(opt::Optimizer, ::MOI.NodeCount)
     return MOI.get(opt.oa_model, MOI.NodeCount())
 end
+
+"""
+    NumberOfCuts
+
+Number of cuts added to the outer approximation model.
+"""
+struct NumberOfCuts <: MOI.AbstractOptimizerAttribute end
+
+function MOI.get(opt::Optimizer, ::NumberOfCuts)
```

Comment:
Implement `is_set_by_optimize` and `attribute_value_type`, you can take example on `MOI.NodeCount` for instance

Reply:
these two methods are not implemented for NodeCount here no?

Reply:
ah ok I saw what you meant, done